openvdb/openvdb/python/README.md

```diff
@@ -0,0 +1,87 @@
+# `openvdb`
+
+This project contains python bindings for OpenVDB.
+
+## Dependencies
+
+| Name    | Version |
+| ---     | ---     |
+| cmake   | >=3.15  |
+| numpy   | >=1.14  |
```

Comment:
This should be defined in in the `pyproject.toml`'s `project.dependencies` attribute.

Reply:
Removing this now :+1: